sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/Sampler.java

```diff
@@ -56,12 +58,14 @@ static Sampler parentBased(Sampler root) {
   }
 
   /**
-   * Returns a {@link ParentBasedSamplerBuilder} that follows the parent's sampling decision if one
-   * exists, otherwise following the root sampler and other optional sampler's decision.
+   * Returns a {@link ParentBasedSamplerBuilder} that enables configuration of the parent-based
+   * sampling strategy. The parent's sampling decision if one exists, otherwise following the root
```

Comment:
> The parent's sampling decision if one exists

It seems like some verb is missing?

Reply:
I re-worked the language in here a bit.

Reply:
sorry...I *will* rework the language a bit.

Reply:
and now I *have* reworked the language.